src/models/City.ts

```diff
@@ -0,0 +1,18 @@
+export class  City{
+    id: String;
+    name: string
+    temperature: string;
+    humidity: string;
+
+    constructor(
+        id: string,
+        name: string,
+        temperature: string,
+        humidity: string
+    ){
+        this.id = id,
```

Comment:
Are you planning to use an integer ID or a UUID

Reply:
I will use an Integer id